src/MediaPlayer/MPNowPlayingInfoCenter.cs

```diff
@@ -138,73 +140,81 @@ void Add (NSMutableDictionary dictionary, NSObject key, NSObject value)
 				dictionary.Add (key, value);
 		}
 
+		bool TryGetValue (NSDictionary source, NSObject? key, [NotNullWhen (true)] out NSObject? result)
+		{
+			if (key != null)
```

Comment:
I don't understand the null check here, it's already done in MPNowPlayingInfo?

Reply:
> it's already done in MPNowPlayingInfo?

where is `key` checked for `null` ?